observability/kiali/ossm-console-plugin.adoc

```diff
@@ -0,0 +1,60 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ossm-console-plugin"]
+= Using OpenShift Service Mesh Console plugin
+include::_attributes/common-attributes.adoc[]
+:context: ossm-console-plugin
+
+toc::[]
+
+The {SMPlugin} extends the {ocp-short-name} web console with a {SMProductShortName} menu and enhanced tabs for workloads and services.
+
+include::modules/ossm-about-console-plugin.adoc[leveloffset=+1]
+
+[id="ossm-install-console-plugin_{context}"]
+== Installing {sm-plugin-full}
+
+You can install the {SMPluginShort} with the Kiali Operator by creating a `OSSMConsole` resource with the corresponding plugin settings. It is recommended to install the latest version of the Kiali Operator, even while installing a previous {SMPluginShort} version, as it includes the latest z-stream release.
+
+.OSSM version compatibility
+[options="header"]
+|===
+|OSSM version |Kiali Server version |OSSMC plugin version |OCP version
+
+|3.0 |v2.4 |v2.4 |4.15+
+
+|2.6 |v1.73 |v1.73 |4.15-4.18
+
+|2.5 |v1.73 |v1.73 |4.14-4.18
+|===
+
+[NOTE]
+====
+{SMPluginShort} is only supported on {ocp-short-name} 4.15 and above. For {ocp-short-name} 4.14 users, only the standalone Kiali console is accessible.
+====
+
+You can install the {SMPluginShort} by using the {ocp-product-title} web console or the {oc-first}.
```

Comment:
Ah - apparently the `oc-first` attribute isn't in the service mesh attributes file. If you want to add it to your file for use, the format is a little weird:

```
:oc-first: pass:quotes[OpenShift CLI (`oc`)]
```

Otherwise, you can just type out the thing when you need it (I find that annoying to type, hence created the attribute :wink:)
```suggestion
You can install the {SMPluginShort} by using the {ocp-product-title} web console or the OpenShift CLI (`oc`).
```

Reply:
Thanks @jeana-redhat! I removed the attribute because it wasn't displaying correctly. I will try adding the attribute and see what happens again. 😄

Reply:
it's working, yay :tada: